src/gui/spim.h

```diff
@@ -168,6 +171,7 @@ public slots:
 
     QList<OrcaFlash *> camList;
     QList<SaveStackWorker *> ssWorkerList;
+    QList<bool> enabledCameras;
```

Comment:
`enabledCameras` makes me think of a list of cameras, whereas this is a list of `bool`. Maybe we can find a better name, like `camEnabled` or `cameraEnabled`. To refactor a symbol in Qt Creator, use Ctrl+Shift+R